functions/go/create-setters/README.md

```diff
@@ -0,0 +1,135 @@
+# create-setters
+
+### Overview
+
+<!--mdtogo:Short-->
+
+Create Setters are the template-free solution to set a comment for matching resource fields.
+
+<!--mdtogo-->
+
+### Synopsis
+
+<!--mdtogo:Long-->
+
+Setters are a safer alternative to other substitution techniques which do not
+have the context of the structured data. Setters may be invoked to modify the
```

Comment:
Setter comments can be added to parameterize the field values of resources using `create-setters` function.